### PR TITLE
Onboard ansible/configs/controller-migration-casc

### PR DESCRIPTION
**Pull request by klewis0928**
##### SUMMARY

Onboarding new config and role 

##### ISSUE TYPE
- New config Pull Request

##### COMPONENT NAME
ansible/configs/controller-migration-casc
ansible/roles/awx_pod/

```

### CODE REVIEW

**Review by bbethell-1 — APPROVED** [review]

File: collections/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+collections:
+  - community.mysql
+...
```
Comment (fridim) [inline collections/requirements.yaml:4]: what is this file?

File: ansible/configs/controller-migration-casc/files/repos_template.j2 [hunk 2]
```diff
@@ -0,0 +1,38 @@
+[rhel-7-server-rpms]
+name=Red Hat Enterprise Linux 7
```
Comment (fridim) [inline ansible/configs/controller-migration-casc/files/repos_template.j2:2]: rhel7? Seems a bit old, is that intended?

File: ansible/configs/controller-migration-casc/README.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+and adapt the variables to your needs. You can also add and overwrite variables
+present in the link:env_vars.yml[env_vars.yml file].
+
+TIP: you'll need to adapt _once_ all variables marked with `CHANGEME` to adapt them to your own person.
+The variables marked with `CHANGEME-ec2` need to be adapted once for each new AWS-sandbox deployment (as long as you keep the same sandbox, they don't need to change).
+The GUID only needs to be different if you deploy the same environment multiple times in the same sandbox.
+
+== Running Ansible Playbook
+
+From the `agnosticd/ansible` directory run:
```
Comment (fridim) [inline ansible/configs/controller-migration-casc/README.adoc:26]: can you specify which Execution Environment should be used here? Is it ee-multicloud?

File: ansible/configs/controller-migration-casc/sample_vars.yml [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+#Define users you want to create. Set superuser: yes to make user system wide System Administrator
+tower_user_accounts:
+- user: babylon
+  password: changeme
+  email: babylon@example.com
+  firstname: Baby
+  lastname: Lon
+  superuser: true
+- user: babylon-viewer
+  password: changeme
```
Comment (fridim) [inline ansible/configs/controller-migration-casc/sample_vars.yml:131]: can this value be left empty in samples and examples and then the roles automatically generate a random password if that's the case?
We've seen too many hardcoded passwords and that becomes a problem

File: ansible/configs/controller-migration-casc/sample_vars.yml [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+  lastname: Viewer
+#   - user: test3
+#   - user: test4
+#     lastname: four
+
+
+tower_credentials:
+- name: babylon-tower-credential
+  username: babylon
+  password: changeme
```
Comment (fridim) [inline ansible/configs/controller-migration-casc/sample_vars.yml:143]: same Q

File: ansible/roles/awx_pod/defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+# ca_trust_file: /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
+
+#
+# anything below is pod internal info, never seen outside.
+#
+pg_hostname: localhost
+pg_port: 5432
+pg_database: awx
+pg_username: awx
+pg_password: awxpass
```
Comment (fridim) [inline ansible/roles/awx_pod/defaults/main.yml:49]: can password default values please be automatically generated instead?

File: collections/requirements.yaml (hunk 1 above)
Comment (ivarmu) [reply collections/requirements.yaml:4]: not used

File: ansible/configs/controller-migration-casc/files/repos_template.j2 (hunk 2 above)
Comment (ivarmu) [reply ansible/configs/controller-migration-casc/files/repos_template.j2:2]: not used

File: ansible/configs/controller-migration-casc/sample_vars.yml (hunk 5 above)
Comment (ivarmu) [reply ansible/configs/controller-migration-casc/sample_vars.yml:143]: This is the `sample_vars.yml` file, that is not used directly... but I've changed it to use the `tower_admin_password` variable, which is autogenerated.

File: ansible/roles/awx_pod/defaults/main.yml (hunk 6 above)
Comment (ivarmu) [reply ansible/roles/awx_pod/defaults/main.yml:49]: changed to use `"{{ awx_admin_password | default('password') }}"` instead

File: collections/requirements.yaml (hunk 1 above)
Comment (ivarmu) [reply collections/requirements.yaml:4]: yeah, It is used finally:

image

**Pull request closed without merging**